<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="pushButtonBackClicked">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pushButtonNextClicked">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="lineEditMessage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="StartPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>470</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>470</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>действие</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ChangedPointPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>470</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>470</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>баллы</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>плюс</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>минус</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ChoicePage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonLab">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Lab</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonCallStudents">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Call students</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonChangedPoint">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Plus/Minus point</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonSaveData">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Save data and end program</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CallPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item alignment="Qt::AlignHCenter">
         <widget class="QTextEdit" name="textEditOutput">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>350</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QTextEdit" name="textEditInput">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LabPage">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="pushButtonCansel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>отменить</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>400</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>500</height>
               </size>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>198</width>
                 <height>498</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayoutNames"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>400</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>500</height>
               </size>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>198</width>
                 <height>498</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayoutLabNames"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonCreateLab">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string> Create</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonChangedLab">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string>Changed</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonDeleteLab">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
